kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -503,11 +501,8 @@
           facility: this.selectedFacility.id,
         };
 
-        if (plugin_data.oidcProviderEnabled) {
+        if (this.nextParam) {
```

Comment:
How does work now with the `plugin_data.oidcProviderEnabled` flag removed? Does the plugin have code that will add the `?next` param?

Reply:
No, it just meant that the redirect would only work if the plugin was enabled.

As this is standardizing us on using `next` for all redirects, there's no need to conditionalize it any more.